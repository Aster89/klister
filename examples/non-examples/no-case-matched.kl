#lang "prelude.kl"

(example
  (case (true)
    [(false)
     42]))
