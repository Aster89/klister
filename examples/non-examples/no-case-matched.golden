Error at phase p0: No case matched at no-case-matched.kl:4.3-6.10: (true)
